alacritty_terminal/src/vi_mode.rs

```diff
@@ -756,4 +746,45 @@ mod tests {
         cursor = cursor.motion(&mut term, ViMotion::WordLeft);
         assert_eq!(cursor.point, Point::new(Line(0), Column(0)));
     }
+
+    fn history_term() -> Term<()> {
+        let mut term = term();
+
+        for i in 0..(term.screen_lines() as i32) {
+            term.grid_mut()[Line(i)][Column(0)].c = 'c';
+        }
+        term.grid_mut().clear_viewport();
+
+        term
+    }
```

Comment:
What's the purpose of this function?

Reply:
As this function name the purpose is to create a terminal which have scrollback buffer. But come to think of it, it is enough to call `clear_viewport()`.